community/sinit/files/sinit.post.shutdown

```diff
@@ -0,0 +1,14 @@
+# We handle the shutdown from here
+
+if [ "$SINIT_SHUTDOWN_HOOKS" = 1 ]; then
+    case "$1" in
+        reboot)	
+            out "Requesting system reboot..."
+            halt -r 
+            ;;
+        *) 
+            out "Requesting system poweroff..."
+            halt -p 
+            ;;
+    esac
+fi
```

Comment:
Why is this needed?

Reply:
If we don't do that system will not poweroff and we will be stranded without a getty and system services on an empty screen.

Reply:
How does busybox init work around this? Does the `shutdown` command not trigger this for us?

Reply:
As far as i know, it is built inside init. System halts when the shutdown functions on the inittab are complete.